packages/workbox-webpack-plugin/src/lib/generate-webpack-manifest.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (manifestEntries, manifestVarName) =>
+  new Promise((resolve) => resolve(
```

Comment:
Nit: You can use `Promise.resolve(value)` as shorthand to create a promise that resolves with a given value. No need to explicitly construct a `Promise` and pass it a function.